.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__
+*.py[cod]
+venv
```

Comment:
shall we not make changes to .gitignore?

Reply:
happy to remove, but why not include it? these are pretty standard entries to gitignore for python projects.

Reply:
i am not an expert on git so i am ok either way. maybe others have some idea of why we have never included these entries from the beginning?

Reply:
i saw we have .gitignore in some sub directories like similar region / docs, not sure why we didn't have it at top level. I guess it doesn't hurt anyway

Reply:
fwiw i later noticed Michael had a proposed change for this as well: https://github.com/gro-intelligence/api-client/pull/171

I'll incorporate the discussion in that PR into a separate PR. Thanks!